core/templates/dev/head/profile/Profile.js

```diff
@@ -28,10 +28,20 @@ oppia.controller('Profile', [
       return oppiaDatetimeFormatter.getLocaleDateString(millisSinceEpoch);
     };
 
+    $scope.areRatingsShown = function(ratingFrequencies) {
+    return ratingVisibilityService.areRatingsShown(ratingFrequencies);
+  }; 
+
     $rootScope.loadingMessage = 'Loading';
     $http.get(profileDataUrl).success(function(data) {
       $rootScope.loadingMessage = '';
       $scope.userBio = data.user_bio;
+      $scope.userDisplayedStatistics = [
+      {'number_value': data.user_impact_score, 'title': "User Impact Score"},
```

Comment:
Intend this and the next two lines by two (since they're inside a '[').

Use single quotes in JS (this is a codebase convention).